model/project_vars.go

```diff
@@ -216,15 +221,36 @@ func (projectVars *ProjectVars) GetRestrictedVars() map[string]string {
 	return restrictedVars
 }
 
-func (projectVars *ProjectVars) GetAdminOnlyVars() map[string]string {
+// GetAdminOnlyVars will change to GetVars on EVG-16045 after removing restricted vars.
+func (projectVars *ProjectVars) GetAdminOnlyVars(t *task.Task) (map[string]string, error) {
```

Comment:
This doesn't return any errors anymore so let's remove it from the function signature

Reply:
done